inception-project/inception
# #1076 - Entity candidate ranking too biased towards query or mention similarity

**What's in the PR**
- Use geometric mean to balance the Levenshtein distances between candidate and query/mention

**How to test manually**
* Basically try linking entities with different mentions and queries and see if the candidate ranking looks particularly bad

**Automatic testing**

**Documentation**
